packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -550,14 +640,20 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
 
   @override
   void goIdle() {
-    beginActivity(_createIdleScrollActivity(_outerPosition), _createIdleScrollActivity);
+    beginActivity(
+      _createIdleScrollActivity(_outerPosition),
+      _createIdleScrollActivity,
+    );
   }
 
   @override
   void goBallistic(double velocity) {
     beginActivity(
       createOuterBallisticScrollActivity(velocity),
-      (_NestedScrollPosition position) => createInnerBallisticScrollActivity(position, velocity),
+      (_NestedScrollPosition position) => createInnerBallisticScrollActivity(
```

Comment:
nit: cannot use => if you turn this into a multi-line method as per style guide.